Hook in select

juwara0 opened the pull request:
# feature

Closes: https://github.com/ciena-frost/ember-frost-core/issues/163
# CHANGELOG
- **Added** hooks for testing into frost-select and frost-multi-select
- **Added** tests to verify hooks in integration tests for frost-select and frost-multi-select
- **Added** documentation on the hook usage for frost-select and frost-multi-select

juwara0 committed:
added ember-hook to frost-select, updated docs and tests

juwara0 committed:
added ember-hook to frost-multi-select and added tests

juwara0 committed:
update docs with additional hook for multi-select

sglanzer-fire commented on docs/frost-select.md:
```diff
@@ -18,6 +18,14 @@
+## Testing with ember-hook
+The select component is accessible using ember-hook with the top level hook name or you can access the internal components as well -
+* Input field hook - `$hook('<hook-name>-input')`
+* List hook - `$hook('<hook-name>-list').find('li')`
```
Do we need `li` here?  The hook was on the `ul` which seems correct

juwara0 replied:
I agree. The hook is for the list. The `<li>` has been removed.

sglanzer-fire commented:
Awesome - ember-hook is seriously the best testing tool since sliced bread ;)

On a side note - is it possible for Reviewable to not be listed in the Changelog section (since it currently ends up in the actual changelog, which seems incorrect)

sglanzer-fire commented:
Approved - just check over the `ul` comment

[![Approved with PullApprove](https://img.shields.io/badge/pullapprove-approved-brightgreen.svg)](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/164/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

juwara0 committed:
update the documentation for the select to refer to the ul tag instead of the li

juwara0 commented:
👍  self approving after addressing the comment to modify the documentation item about how to access the hook for the list.

[![Approved with PullApprove](https://img.shields.io/badge/pullapprove-approved-brightgreen.svg)](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/164/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)
